craft_cli/messages.py

```diff
@@ -781,6 +781,29 @@ def set_secrets(self, secrets: list[str]) -> None:
         """Set the list of strings that should be masked out in all output."""
         self._printer.set_secrets(secrets)
 
+    @_active_guard()
+    def confirm(self, prompt: str, *, default: bool = False) -> bool:
```

Comment:
is there any change here from the charmcraft version? Other than the `self.pause()`